### DISCUSSION TITLE
Include Watchlist in PM4K?

### DISCUSSION THREAD

**Discussion by 4KFTW** [pannal/plex-for-kodi#162] [Q&A] [answered]
Is is possible to include the Watchlist in Plex into PM4K? Thanks.

**Comment by pannal** [comment, marked as answer]
Yes. It's planned for the 0.8.0 release.

**Comment by 4KFTW** [comment]
Great! Can't wait! I use Watchlist heavily, so really need this to use PM4K.

**Reply by pannal** [reply]
I know. It's been requested for ages :)

**Reply by tundrablasen** [reply]
Is there any progress or ETA on the PLEX Watchlist @pannal 

Thanks for your work on PM4K

**Reply by pannal** [reply]
We're at about 80% of the basic implementation. Availability checks and add/remove from watchlist is still open. You can keep track here: https://github.com/pannal/plex-for-kodi/tree/watchlist

**Reply by pannal** [reply]
Basic implementation is done and included in the latest beta.

**Reply by tundrablasen** [reply]
That is great news. Thanks!!
